lib/ci-stack.ts

```diff
@@ -96,6 +96,7 @@ export class CIStack extends Stack {
 
     const auditloggingS3Bucket = new CiAuditLogging(this);
     const vpc = new Vpc(this, 'JenkinsVPC', {
+      availabilityZones: ['us-east-1a', 'us-east-1b', 'us-east-1c'],
```

Comment:
This should be dynamic. I always use this repo to deploy in us-west-2. In that case, the deployment would fail. Either take this as context parameters or keep whatever is default.

Reply:
Without this change VPC is getting recreated. Hardcodes as both prod and beta deploy in us-east-1. Passing the values thorough context parameter can be implemented.

Reply:
Yeah that is just for https://build.ci.opensearch.org/. Take into consideration for users deploying in their own dev environment.